Sprint-2/implement/querystring.test.js

```diff
@@ -3,10 +3,28 @@
 // Below is one test case for an edge case the implementation doesn't handle well.
 // Fix the implementation for this test, and try to think of as many other edge cases as possible - write tests and fix those too.
 
-const parseQueryString = require("./querystring.js")
+const parseQueryString = require("./querystring.js");
 
 test("parses querystring values containing =", () => {
-  expect(parseQueryString("equation=x=y+1")).toEqual({
-    "equation": "x=y+1",
+  
+    expect(parseQueryString("equation=x=y+1")).toStrictEqual({
+      equation: "x=y+1",
+    });
   });
-});
+  test("parses querystring ", () => {
+    expect(parseQueryString("string1=x+1&string2=x-1")).toStrictEqual({
+      string1: "x+1",
+      string2: "x-1",
+    });
+  });
+  test("an empty querystring should return {} ", () => {
+    expect(parseQueryString("")).toStrictEqual({});
+  });
+  test("parses querystring with empty value", () => {
+    expect(parseQueryString("string1=x+1&string2=")).toStrictEqual({
+      string1: "x+1",
+      string2: "",
+    });
+  });
+  
```

Comment:
For each of the following function calls, does your function return the value you expected?
```
parseQueryString("a=b&=&c=d")
parseQueryString("a=")
parseQueryString("=b")
parseQueryString("a=b&&c=d")
parseQueryString("a%25b=c%26d`)
```
Note: the `%25` and `%26` are ***URL encoded*** or ***percent encoded*** characters.